### DISCUSSION TITLE
Translate "Gyawun Music" into another language

### DISCUSSION THREAD

**Discussion by Mickael81** [sheikhhaziq/gyawun_music#195] [Q&A]
Hello,
I have already managed to translate other streaming applications published on GitHub and which use Weblate and Crowdin for translation. I would have liked to be able to do the same with "Gyawun Music".
If you agree, could you tell me the link and the procedure to follow to be able to translate the application into French?
Thank you very much for your help!
Kind regards

**Comment by sheikhhaziq** [comment]
I am working on an new major update which may break some some things. Adding language before that may be useless. When the development is near completion, I will let you know so that you can add French language to it.

**Comment by Mickael81** [comment]
All right! Thank you very much for your feedback and for these explanations. 🙂👍

**Comment by Mickael81** [comment]
@sheikhhaziq I tried to make a pull request but apparently it was denied. As you told me last month, when development is almost finished, let me know so I can contribute to the French translation as I have done for other applications on GitHub.
![Screenshot_20250108_201507_Chrome](https://github.com/user-attachments/assets/883e4fda-811d-468f-9a81-1efc94d65349)
